sass/base/components/card-block/_card.scss

```diff
@@ -6,18 +6,16 @@
 //
 // Style guide: 20.60
 
-.cards__item {
+.card {
   position: relative;
   display: inline-block;
   overflow: hidden;
   margin-bottom: 20px;
   text-align: left;
-  @include breakpoint($screen-md-only) {
-    width: calc(100% - (4 * 1.25%)) !important;
-    margin: 1.25% calc(2 * 1.25%) !important;
-  }
+  width: calc(100% - (4 * 1.25%));
```

Comment:
Maybe nicer to use the grid/gutter variables here, rather than hardcoded % values?